Rakefile

```diff
@@ -36,4 +36,22 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task default: [:test, :rubocop]
+task default: [:test, 'rubocop:auto_correct']
```

Comment:
So, I always run with auto_correct locally since if there's a problem, I would just run it next.  then, I review the changes and either accept them, filter them, or change some config.  I don't see any advantage to running rubocop by default in dev but not auto-correcting.

So, I feel pretty strongly that if we're going to force devs to opt out of rubocop when testing locally, to at least let the rubocop help.  Other repos just remind you what the dev process is, e.g. https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md#running-rubocop